content/fr/docs/setup/_index.md

```diff
@@ -0,0 +1,93 @@
+---
+reviewers:
```

Comment:
I'm not sure about that, but I think that for this line we have to change the names by ours.

Reply:
@zacharysarah could you help us here understand if we have to change that? thanks

Reply:
I agree, this should point to OWNERS of sig-docs-fr

Reply:
done with current reviewer+remy